Resources/Storage/SRM2Storage.py

```diff
@@ -69,7 +69,7 @@ def __init__( self, storageName, parameters ):
     self.gfalRetry = gConfig.getValue( "/Resources/StorageElements/GFAL_Retry", 3 )
 
     # # set checksum type, by default this is 0 (GFAL_CKSM_NONE)
-    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", None )
+    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", '' )
```

Comment:
Yes it harms, because there is a lookup done after. Why would "pylon" complain ? :D